.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
@c1rrus can you check if this works in Windows (without the Linux subsystem)?

Reply:
I can, but it might be a while before I find the time to set it all up (got quite a lot going on at work and at home right now). I therefore suggest not holding up this PR. If I find issues when I eventually get around to trying it, I can always open a follow-up PR to address them then.

Reply:
According to their docs, it works in Windows (as long as one has installed the appropriate tooling).